www/class/centreonAuth.SSO.class.php

```diff
@@ -365,23 +369,28 @@ public function getOpenIdConnectToken(
         string $clientId,
         string $clientSecret,
         string $code,
+        bool $clientBasicAuth,
```

Comment:
can you also modify function definition?

Reply:
Sorry, added the function documentation for this parameter